flume-ng-core/src/test/java/org/apache/flume/source/TestAvroSource.java

```diff
@@ -97,20 +96,19 @@ public void testLifecycle() throws InterruptedException {
     boolean bound = false;
 
     for (int i = 0; i < 100 && !bound; i++) {
-      try {
-        Context context = new Context();
-
-        context.put("port", String.valueOf(selectedPort = 41414 + i));
-        context.put("bind", "0.0.0.0");
 
-        Configurables.configure(source, context);
+      Context context = new Context();
 
+      context.put("port", String.valueOf(selectedPort = 41414 + i));
+      context.put("bind", "0.0.0.0");
+      // Invalid configuration may throw a FlumeException which has to be expected in the callers
```

Comment:
With one exception I removed this comment. In that occurrence the helper function is called by different tests and some of the tests expect a FlumeException to be thrown caused by invalid configuration. It could have saved me some time during implementation that `configure()` can also raise a FlumeException so I would like to make life of future devs a bit easier. Also I rephrased a little the comment.